CHANGELOG.md

```diff
@@ -60,6 +61,17 @@ Adding a new version? You'll need three changes:
  - [0.0.5](#005)
  - [0.0.4 and prior](#004-and-prior)
 
+## [2.8.0]
```

Comment:
This will render an incorrect markdown heading: `[2.8.0]` without the hyper link. I suggest we drop the square brackets until the 2.8.0 is ready.

The other option would be to add the following link definition at the bottom of the changelog

```
[2.8.0]: https://github.com/kong/kubernetes-ingress-controller/compare/v2.7.0...main
```

which would not be 100% true.

Reply:
I usually just add the eventual link (`https://github.com/kong/kubernetes-ingress-controller/compare/v2.7.0...v2.8.0`) and let it be a 404 until it's out. We tend to forget it if we don't do it in whatever change set originally adds it. They're broken, sure, but I figure most people aren't clicking around much anything for the unreleased versions (the broken link hasn't generated questions in the past).